source/fundamentals/indexes.txt

```diff
@@ -393,7 +393,7 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
       E11000 duplicate key error index
 
-Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
+Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB Server manual for more information.
```

Comment:
[suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".